### PR TITLE
CC for "Getting an RSE org started" doc

### LINKED ISSUE #1999: CC for "Getting an RSE org started" doc
Author: @ksbeattie 
EB member: @rinkug 

A first CC for me on: 
[Getting Started with the RSE Movement within your Organization: A Guide for Individuals](https://zenodo.org/records/10436166)

Targeted for March

### PR DESCRIPTION
**Pull request by ksbeattie**
# Description
My CC for the "Getting Started with the RSE Movement within your Organization: A Guide for Individuals" document [here](https://zenodo.org/records/10436166).

EB Member: @pagrubel 

Resolves #1999

## PR checklist for files displayed on bssw.io site

* [x] ***[Author]*** `@mention` the BSSw.io editorial board member `@<eb-member-id>` in **Description** above assigned to shepherd your PR.
* [x] ***[Author]*** Add the `<issue-id>` in the **Description** above for the associated GitHub Issue.
* ~~[ ] ***[Author]*** Ensure `wikize_refs.py -i <base>.md` is run and commit (if using [wikize_refs.py])~~.
* [x] ***[Author]*** Inspect the content in the `*.md` file(s) as rendered in GitHub for this PR.
* [x] ***[EB Mem]*** Assign this PR to the EB member `<eb-member-id>`.
* [x] ***[EB Mem]*** Assign this PR to the author of the PR `<pr-author-id>`.
* [x] ***[EB Mem]*** Add one or more Reviewers.
* [x] ***[EB Mem]*** Add label `content: <content-type>` for the type of contribution.
* [x] ***[EB Mem]*** Add to Project `Content Development` (see [Content Development]).
* [x] ***[EB Mem]*** Add [meta-data] to the `*.md` file(s) (set `Publish: yes`).
* [x] ***[EB Mem]*** Add label `preview` (so PR branch will be merged to 'preview' branch and watch for possible merge failures).
* [x] ***[EB Mem]*** Rebuild [preview] site and confirm new content is there, renders correctly and is returned in searches.
* [x] ***[Author]*** Make any final changes to the PR based on feedback.
* ~~[ ] ***[Author]*** Ensure `wikize_refs.py -i <base>.md` is run and commit (if using [wikize_refs.py]).~~
* [x] ***[EB Mem]*** Rebuild [preview] site and re-confirm content looks correct.
* [x] ***[EB Mem]*** Ensure at least one reviewer signs off on the final changes.
* [x] ***[EB Mem]*** Move the PR to "Ready to Publish" in [Content Development].
* [ ] ***[EB Mem]*** Leave comment and @mention `@betterscientificsoftware/bssw-maint` (**BSSw Maint**) asking to carry out final publication steps.
* [ ] ***[BSSw Maint]*** Verify that all needed files are present in the PR (article, images, updates to Site/Homepage.md carousel and/or Site/Announcements/Announcements.md as appropriate).
* [ ] ***[BSSw Maint]*** Merge PR. (Should automatically move to "Done" in [Content Development].)
* [ ] ***[BSSw Maint]*** Verify new contribution shows up on [bssw.io] as expected.

NOTE:
* Checklist items prefixed with ***[Author]*** are expected to be performed by the author of the PR or can be performed by the author.
* Checklist items prefixed with ***[EB Mem]*** must be performed by a [BSSw.io Editorial Board (EB) Member](https://betterscientificsoftware.github.io/bssw.io/bssw_members.html).
* Checklist items prefixed with ***[BSSw Maint]*** must be performed by someone in the `@betterscientificsoftware/bssw-maint` team (hint: type `@`, `b`, `s`, `s`, `w`, `-`, `m`  to auto-complete to `@betterscientificsoftware/bssw-maint`).

[preview]: https://preview.bssw.io
[bssw.io]: https://bssw.io
[Content Development]: https://github.com/betterscientificsoftware/bssw.io/projects/3
[BSSw Internal]: https://github.com/betterscientificsoftware/bssw.io/projects/2
[meta-data]: https://betterscientificsoftware.github.io/bssw.io/bssw_styling_common.html#metadata-section
[wikize_refs.py]: https://github.com/betterscientificsoftware/bssw.io/blob/master/utils/README.md#wikize_refspy

### CODE REVIEW

**Comment by pagrubel** [comment]
@ksbeattie Do you have a draft for this?

**Comment by ksbeattie** [comment]
> @ksbeattie Do you have a draft for this?

I do now, just updated this PR and took it out of draft.  Please review and provide feedback!

File: CuratedContent/GettingStartedWithAnRSEMovement.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+:--- | :---
+Article title  | Getting Started with the RSE Movement within your Organization: A Guide for Individuals
+Website |  https://zenodo.org/records/10436166
+Focus | Community Building
+
+Have you ever thought about starting an RSE movement within your university, lab or company?
+Struggled with how exactly you might go about getting one started and the challenges you might face?
+The article [Getting Started with the RSE Movement within your Organization: A Guide for
+Individuals](https://zenodo.org/records/10436166) provides a detailed exploration of exactly these
+issues by providing practicle steps and tips for creating a local community of RSEs.  Starting with
```
Comment (pagrubel) [inline CuratedContent/GettingStartedWithAnRSEMovement.md:21]:
```suggestion
issues by providing practical steps and tips for creating a local community of RSEs.  Starting with
```

File: CuratedContent/GettingStartedWithAnRSEMovement.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+how to raise awareness, creating spaces to participate together, recruting allies and formalizing
+the new group, this article goes into details and examples on how to promote the RSE role and
+advocate for individluals to impact changes both at the personal and institutional level.
```
Comment (pagrubel) [inline CuratedContent/GettingStartedWithAnRSEMovement.md:24]:
```suggestion
advocate for individuals to impact changes both at the personal and institutional level.
```

File: CuratedContent/GettingStartedWithAnRSEMovement.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+#### Publication date:  March 31, 2024
+
+ 
+Resource information | Details
+:--- | :---
+Article title  | Getting Started with the RSE Movement within your Organization: A Guide for Individuals
+Website |  https://zenodo.org/records/10436166
+Focus | Community Building
+
+Have you ever thought about starting an RSE movement within your university, lab or company?
```
Comment (pagrubel) [inline CuratedContent/GettingStartedWithAnRSEMovement.md:17]:
```suggestion
Have you ever thought about starting an RSE movement within your university, lab, or company?
```

File: CuratedContent/GettingStartedWithAnRSEMovement.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+Struggled with how exactly you might go about getting one started and the challenges you might face?
+The article [Getting Started with the RSE Movement within your Organization: A Guide for
+Individuals](https://zenodo.org/records/10436166) provides a detailed exploration of exactly these
+issues by providing practicle steps and tips for creating a local community of RSEs.  Starting with
+how to raise awareness, creating spaces to participate together, recruting allies and formalizing
```
Comment (pagrubel) [inline CuratedContent/GettingStartedWithAnRSEMovement.md:22]:
```suggestion
how to raise awareness, creating spaces to participate together, recruiting allies and formalizing
```

**Review by pagrubel — CHANGES_REQUESTED** [review]
@ksbeattie This looks good, I've requested some fixes to typos that can be incorporated.

**Comment by pagrubel** [comment]
@ksbeattie I went ahead and committed those small changes and the preview is rebuilding please check it and make sure it's ok for you. I'll approve when you are happy with it. I also put it on the content board.

**Review by pagrubel — APPROVED** [review]

**Comment by ksbeattie** [comment]
I'm happy with all the spelling corrections - thank you!

**Pull request merged**